docs/asciidoc/modules/ROOT/pages/overview/apoc.uuid/apoc.uuid.create.adoc

```diff
@@ -0,0 +1,44 @@
+////
+This file is generated by DocsTest, so don't change it!
+////
+
+= apoc.uuid.create
+:description: This section contains reference documentation for the apoc.uuid.create procedure.
+
+label:procedure[] label:apoc-core[]
+
+[.emphasis]
+CALL apoc.uuid.install(databaseName, label, $config) | eventually adds the uuid transaction handler for the provided `label` and `uuidProperty`.
+In case the UUID handler is already present it will be replaced by the new one
+
+== Signature
+
+[source]
+----
+apoc.uuid.create(databaseName :: STRING?, label :: STRING?, config = {} :: MAP?) :: (batchComputationResult :: MAP?, label :: STRING?, installed :: BOOLEAN?, properties :: MAP?)
```

Comment:
couldn't we just have added database as the last parameter with a default value?
or alternatively added a `database` entry to the config?